src/jobs/keys-update/staking-module-updater.service.ts

```diff
@@ -31,6 +31,7 @@ export class StakingModuleUpdaterService {
       isReorgDetected: false,
     };
 
+    // TODO: Promise.all
```

Comment:
No need to process them in parallel, it was originally written that way and we had problems with memory consumption